students/aaron/lesson02/assignment/src/charges_calc.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+def setup_logger(level):
+    """
+    Sets up the logger if desired, or a dummy logger by default.
+    """
+    logger = logging.getLogger(__name__)
+    # return the disabled logger if no logging requested
+    if level is None:
+        logger.disabled = True
+        return logger
+    log_format = " ".join(["%(asctime)s",
+                           "%(filename)s:%(lineno)-3d",
+                           "%(levelname)s %(message)s"])
+    log_file = datetime.datetime.now().strftime("%Y-%m-%d")+'.log'
+    formatter = logging.Formatter(log_format)
+    logger.setLevel(level)
+    file_level = logging.WARNING if level == logging.DEBUG else level
```

Comment:
What is the purpose of this? 
I like how you use this if command.

Reply:
This makes sure that we meet the requirement of not permitting DEBUG messages to be logged to file.  If our overall logging level is DEBUG, then it sets the file logging level to WARNING instead.  Otherwise, the file logging level matches the console logging level.